### PR TITLE
[FEATURE] Checker for correct Python code snippets in documentation.

### PR DESCRIPTION
**Pull request by alexsherstinsky**
### Scope
* Python program is inserted into Azure Continuous Integration pipeline in order to ensure that correct style of including `Python` code snippets in documentation is being adhered to.
* Fix to use correct starting directory for "no line number snippet" checker as well.

### Remarks
* JIRA: DX-338/DX-345
* 

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted

    ```
    black .

    ruff . --fix
    ```
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: ci/checks/check_only_name_tag_snippets.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+import pathlib
+import re
+import shutil
+import subprocess
+import sys
+from typing import List
+
+ITEMS_IGNORED_FROM_NAME_TAG_SNIPPET_CHECKER = {
+    "docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_postgresql_data.md",
```
Comment (alexsherstinsky) [inline ci/checks/check_only_name_tag_snippets.py:46]: These lines should be deleted once the implementation is completed and merged.  This effort is Work-In-Progress and is being tracked via the JIRA ticket indicated in the description.

File: ci/checks/check_no_line_number_snippets.py [hunk 2]
```diff
@@ -8,6 +8,15 @@
 import sys
 from typing import List
 
+ITEMS_IGNORED_FROM_LINE_NUMBER_SNIPPET_CHECKER = {
+    "docs/docusaurus/node_modules/prism-react-renderer/README.md",
```
Comment (anthonyburdi) [inline ci/checks/check_no_line_number_snippets.py:12]: Just a suggestion, could we ignore the whole `/node_modules/` folder? These are packages installed by npm, not our code.

**Review by anthonyburdi — APPROVED** [review]
LGTM! Just one small suggestion (and pending tweaking the excludes to fix the failing test).

File: ci/checks/check_no_line_number_snippets.py (hunk 2 above)
Comment (alexsherstinsky) [reply ci/checks/check_no_line_number_snippets.py:12]: @anthonyburdi Oops! Forgot that one here.  Will do an update PR real quick!  Thanks.

**Pull request merged**